tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -90,6 +91,9 @@ def create(self,
         if background is not None:
             params["background"] = background
 
+        if user_context is not UNDEFINED:
+            params["userContext"] = user_context
+
```

Comment:
Note that we haven't got to agreement on https://github.com/w3c/webdriver-bidi/issues/642 yet